### ISSUE TITLE
[Bug]han.woff2 failed

### ISSUE THREAD

**Issue by Richard-Cao** [iissnan/hexo-theme-next#1848]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

Request complete.

### Actual behavior 

![image](https://user-images.githubusercontent.com/5953992/30043728-be527a3c-922b-11e7-9f00-f33b00d9896e.png)

### Steps to reproduce the behavior 
1. change `han` to true

* Link to demo site with this issue: https://richardcao.me/
* Link(s) to source code or any usefull link(s): N/A

### NexT Information

**NexT Version:**

```
[x] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[x] Pisces
[] Gemini
```

**Comment by ivan-nginx** [comment]
@Richard-Cao file `https://richardcao.me/lib/Han/dist/font/han.woff2` not exists in your `public` directory. Check it and check your Han settings.

**Comment by Richard-Cao** [comment]
I only change `han` to true in _config.yml.
Maybe I need run
```
npm install --save han-css
```
?

**Comment by Richard-Cao** [comment]
But in this pr https://github.com/iissnan/hexo-theme-next/pull/1598, I see `han-css` in it.
[Han official repo](https://github.com/ethantw/Han), **han.woff2** in the font folder.
How to solve?

**Comment by ivan-nginx** [comment]
I don't think what u need to install anything. U can search [Han pull requests](https://github.com/iissnan/hexo-theme-next/pulls?utf8=%E2%9C%93&q=is%3Apr%20han), mb it will help.

Yeah. But `han.woff2` is not present in `source/lib/Han/dist/font` directory. There is only `han.woff`.

**Comment by ivan-nginx** [comment]
@Richard-Cao ok, i think need to simple take `han.woff2` and put it in `source/lib/Han/dist/font` directory.

**Comment by ivan-nginx** [comment]
@Richard-Cao fixed in `000cd69`.
If solved, close it.

**Comment by Richard-Cao** [comment]
Fixed! 
Thank you very much ~
